### DISCUSSION TITLE
Error 104 Connection reset by peer after Openshift upgrade version

### DISCUSSION THREAD

**Discussion by jrosal06** [Kong/kong#13408] [Help]
Hi team,

We have Kong version 2.5 installed as a Gateway API in an OpenShift cluster. Last week we made a version update from 4.12 to 4.14 and from that moment on we observed this message:

`2024/07/22 16:04:30 [info] 1110#0: *116558 client 10.124.15.4 closed keepalive connection (104: Connection reset by peer)`

Many times in the kong pod that works as api gateway, we have the 2.5 image deployed in this cluster.

Could you help us with this "issue"?

**Comment by nowNick** [comment]
Hi @jrosal06 

It's difficult to say what might be causing this issue - this Kong version is very old. Could you try to upgrade Kong? Also could you share a little bit more about your Kong setup? What plugins do you use?
